recipes/medicc2/meta.yaml

```diff
@@ -19,15 +19,15 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler("cxx") }}
   host:
-    - python >=3.6
+    - python=3.9
     - pip
     - numpy>=1.20.1
     - openfst>=1.8.1
     - setuptools
     - cython>=0.29.21
     - pathlib
   run:
-    - python >=3.6
+    - python=3.9
```

Comment:
Is this pin necessary? The readme says
> Python version 3.9 and lower are supported

It would be better to keep this flexible if at all possible.